src/cdk/observers/observe-content.ts

```diff
@@ -35,6 +33,56 @@ export class MutationObserverFactory {
   }
 }
 
+
+/** A factory that creates ContentObservers. */
+@Injectable({providedIn: 'root'})
+export class ContentObserverFactory {
+  constructor(private _mutationObserverFactory: MutationObserverFactory, private _ngZone: NgZone) {}
+
+  create(element: Element, debounce?: number) {
```

Comment:
Shouldn't we call this one `observe`?

Reply:
This doesn't actually start observing, it just creates the observer